storage/raft_transport.go

```diff
@@ -133,7 +133,9 @@ func (t *RaftTransport) RaftMessage(stream MultiRaft_RaftMessageServer) (err err
 					t.mu.Unlock()
 
 					if !ok {
-						return errors.Errorf("Unable to proxy message to node: %d", req.Message.To)
+						return errors.Errorf(
+							"unable to proxy message to node %d: store %d not registered",
```

Comment:
It's definitely a replicaID. Use `req.FromReplica.NodeID`.


Arjun Narayan

On Wed, Jul 13, 2016 at 9:48 AM, Tamir Duberstein <notifications@github.com>
wrote:

> In storage/raft_transport.go
> <https://github.com/cockroachdb/cockroach/pull/7797#discussion_r70627613>:
>
> > @@ -133,7 +133,9 @@ func (t *RaftTransport) RaftMessage(stream MultiRaft_RaftMessageServer) (err err
> >  					t.mu.Unlock()
> >
> >  					if !ok {
> > -						return errors.Errorf("Unable to proxy message to node: %d", req.Message.To)
> > +						return errors.Errorf(
> > +							"unable to proxy message to node %d: store %d not registered",
>
> is req.Message.To even a node? I think it's a replica.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/7797/files/b60e1d43401bf36bb3a7811940ef6d7008affd4d#r70627613>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABfkicuyW0qMXMCpJCpYYg8nZMHYsXU6ks5qVOyhgaJpZM4JLWm6>
> .
>